src/devices/Common/Iot/Device/Multiplexing/Multiplexing.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Use `<TargetFrameworks>netcoreapp2.1;net5.0</TargetFrameworks>`

Reply:
Actually, I think you can drop this .csproj entirely. There's a project file in src/devices/Common that should include all files from all its subfolders.

Reply:
Deleted project.